src/hera.cpp

```diff
@@ -215,7 +262,7 @@ evmc_result hera_execute(
     heraAssert(rev == EVMC_BYZANTIUM, "Only Byzantium supported.");
     heraAssert(msg->gas >= 0, "EVMC supplied negative startgas");
 
-    bool meterInterfaceGas = true;
+    bool meterInterfaceGas = hera->meter_interface;
```

Comment:
Instead of this, just uncomment these lines:
```
      case hera_evm1mode::evm2wasm_contract:
        run_code = evm2wasm(context, run_code);
        ensureCondition(run_code.size() > 8, ContractValidationFailure, "Transcompiling via evm2wasm failed");
        // TODO: enable this once evm2wasm does metering of interfaces
        // meterInterfaceGas = false;
break;
```

Reply:
Would it be okay to leave this as an evmc flag? it'd be easier to both with interface metering enabled/disabled